mailpoet/assets/js/src/newsletter_editor/behaviors/MediaManagerBehavior.js

```diff
@@ -63,6 +63,13 @@ BL.MediaManagerBehavior = Marionette.Behavior.extend({
 
     MediaManager = window.wp.media.view.MediaFrame.Select.extend({
       initialize: function () {
+        /**
+         * As of importing @wordpress/i18n, and since it's using lodash, we need to reset window._ back to
+         * underscore for the media manage to work.
+         * The media manager is using _.each and changes the context using the 3 parameter, however _.each of
+         * lodash doesn't accept a third parameter.
+         */
+        window._ = _;
```

Comment:
This issue is caused not by using `underscore` by itself, but by exposing the module as `window._` in the webpack config, using the `expose-loader` plugin. I'm curious why MailPoet is doing this, instead of just using underscore internally, without making it visible outside its webpack bundle?

Also, `underscore` is a standart part of WordPress. MailPoet could externalize its usages, and specify it as a dependency when registering MailPoet scripts.

Reply:
Since version 4.17.0, `@wordpress/i18n` doesn't depend on Lodash any more. By upgrading in the `package.json`, from the currently used 4.14.0, you could eliminate this Lodash issue completely.

Reply:
As we have another [PR](https://github.com/mailpoet/mailpoet/pull/4786) taking care of the dependencies update, I opted to leave this update to that PR and then adjust after that PR is merged to trunk in a follow up PR, specially since we have 2 instances of such a workaround.

Reply:
@samnajian @jsnajdr The [Gutenberg update PR](https://github.com/mailpoet/mailpoet/pull/4786) was already merged so let's try to remove that workaround.

Reply:
@jsnajdr Thanks for the details!

> I'm curious why MailPoet is doing this, instead of just using underscore internally, without making it visible outside its webpack bundle?

Good question. We'll need to investigate, it's probably been like that for ages.

> Also, underscore is a standart part of WordPress. MailPoet could externalize its usages, and specify it as a dependency when registering MailPoet scripts.

Makes sense, I think with underscore, this would work well. We're not quite there yet to externalize all of our `@wordpress` deps but getting closer by updating our deps and adopting better WP/WC version support policies (L-2 or even L-1) where we could reasonably cover them with test automation.

Reply:
> @samnajian @jsnajdr The [Gutenberg update PR](https://github.com/mailpoet/mailpoet/pull/4786) was already merged so let's try to remove that workaround.

I'd do it in a separate PR, and base it on this branch, thanks for the heads up.

Reply:
@samnajian Sure, anything you prefer. Just unfortunately, merging of the upgrade caused conflicts in this PR.